Armstrong_Number/Armstrong_Number.kt

```diff
@@ -0,0 +1,50 @@
+//Function to get Number of Digits in given Number
+fun order(Num: Int):Int {
+    var count = 0
+    var num = Num
+    while(num > 0) {
+        num /= 10
+        count += 1
+    }
+    return count
+}
+
+//Function to check Armstrong Number
+fun armstrong(Num: Int, digits: Int):Int {
+    var sum = 0
+    var num = Num
+    while(num > 0) {
+        var digit =  num % 10
```

Comment:
you have an extra space after =. Please remove it.

Reply:
Done